type-definitions/most.d.ts

```diff
@@ -44,7 +44,10 @@ export interface Stream<A> {
   concatMap<B>(f: (a: A) => Stream<B>): Stream<B>;
   mergeConcurrently<B>(concurrency: number): Stream<B>;
   merge(...ss: Array<Stream<A>>): Stream<A>;
+  mergeArray(streams: Array<Stream<A>>): Stream<A>;
   combine<B>(f: (a: A, ...args: Array<any>) => B, ...ss: Array<Stream<any>>): Stream<B>;
+  combineArray<B>(f: (...args: Array<any>) => B, streams: Array<Stream<any>>): Stream<B>;
```

Comment:
Should we define a bunch of polymorphic variants for combine and combineArray, [similar to what Rx 5 has done](https://github.com/ReactiveX/rxjs/blob/master/src/operator/combineLatest.ts#L70)?  It's visually awful, but provides some extra type safety when the argument list (or array) is small.